modules/ROOT/nav.adoc

```diff
@@ -15,6 +15,7 @@
 ** xref:configuration:templating.adoc[]
 ** xref:configuration:dynamic_inputs_and_outputs.adoc[]
 ** xref:configuration:using_cue.adoc[]
+** xref:configuration:telemetry.adoc[]
```

Comment:
I've put this topic under configuration. Does this seem right to you?

Reply:
Yeah that's fine with me